### PR TITLE
[native] Fix test data path in Presto-on-Spark native runner

### PR DESCRIPTION
**Pull request by miaoever**
The PR makes sure both actual runner and expected runner share the same test data path when the DATA_DIR arguments is not provided.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java [hunk 1]
```diff
@@ -60,7 +60,8 @@ public void testAggregations()
     @Test
     public void testJoins()
     {
-        assertQuery("SELECT count(*) FROM orders o, lineitem l WHERE o.orderkey = l.orderkey AND o.orderkey % 2 = 1");
+        // assertQuery("SELECT count(*) FROM orders o, lineitem l WHERE o.orderkey = l.orderkey AND o.orderkey % 2 = 1");
```
Comment (mbasmanova) [inline presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java:63]: Accidental change?

**Review by mbasmanova — COMMENTED** [review]
@miaoever I'm curious how did it work before? I assume CI doesn't specify DATA_DIR. Is this test no running in CI?

**Comment by miaoever** [comment]
> DATA_DIR

I believe CI always passed the DATA_DIR - https://github.com/prestodb/presto/blob/master/.circleci/config.yml#L98

**Comment by mbasmanova** [comment]
> @mbasmanova I believe CI always passed the DATA_DIR - https://github.com/prestodb/presto/blob/master/.circleci/config.yml#L98

Got it. I didn't realize this is the case. Thank you for explaining.

File: presto-native-execution/src/test/java/com/facebook/presto/spark/AbstractTestPrestoSparkQueries.java [hunk 2]
```diff
@@ -58,11 +71,12 @@ protected QueryRunner createQueryRunner()
         if (System.getProperty("NATIVE_PORT") == null) {
             String path = requireNonNull(System.getProperty("PRESTO_SERVER"),
                     "Native worker binary path is missing. " +
-                    "Add -DPRESTO_SERVER=/path/to/native/process/bin to your JVM arguments.");
+                            "Add -DPRESTO_SERVER=/path/to/native/process/bin to your JVM arguments.");
```
Comment (mbasmanova) [inline presto-native-execution/src/test/java/com/facebook/presto/spark/AbstractTestPrestoSparkQueries.java:74]: Accidental change?

**Review by mbasmanova — APPROVED** [review]
@miaoever Thank you for the fix. Please, update commit message to fix some typos.

> Make sure both actual and expected runners share the ...

**Comment by ankurpathela** [comment]
> I believe CI always passed the DATA_DIR - https://github.com/prestodb/presto/blob/master/.circleci/config.yml#L98

I also noted this in https://github.com/prestodb/presto/blob/master/.github/workflows/test-native.yml#L107 earlier.

Thank you for the fix @miaoever!

**Pull request merged**